Framework/CurveFitting/inc/MantidCurveFitting/RalNlls/TrustRegion.h

```diff
@@ -19,19 +19,19 @@ void mult_Jt(const DoubleFortranMatrix &J, const DoubleFortranVector &x,
 double evaluate_model(const DoubleFortranVector &f,
                       const DoubleFortranMatrix &J,
                       const DoubleFortranMatrix &hf,
-                      const DoubleFortranVector &d, const nlls_options options,
+                      const DoubleFortranVector &d, const nlls_options &options,
                       evaluate_model_work &w);
 double calculate_rho(double normf, double normfnew, double md,
                      const nlls_options &options);
 void update_trust_region_radius(double &rho, const nlls_options &options,
                                 nlls_inform &inform, NLLS_workspace &w);
-void rank_one_update(DoubleFortranMatrix &hf, NLLS_workspace w);
+void rank_one_update(DoubleFortranMatrix &hf, NLLS_workspace &w);
```

Comment:
I think this affected only performance. `NLLS_workspace` contains pre-allocated helper objects: matrices and vectors and is passed to various functions just to provide temporaries. It's a usual fortran way of doing things. I haven't figured it out yet how to make it the C++ way.